tests/automated_install.sh

```diff
@@ -38,6 +43,18 @@ $SEDINPLACE 's/vb.gui = true/vb.gui = false/' Vagrantfile
 $SEDINPLACE "s/\"dns_servers\" : \[ \"8.8.8.8\", \"8.8.4.4\" \]/\"dns_servers\" : \[ $DNS_SERVERS \]/" environments/${ENVIRONMENT}.json
 [[ -n "$PROXY" ]] && $SEDINPLACE -e "s#\(\"bootstrap\": {\)#\1\\\n\"proxy\" : \"http://$PROXY\",#" -e $'s/\\\\n/\\\n/' environments/${ENVIRONMENT}.json
 
+# download dependent/third-party cookbooks: begin
+[[ -n "$PROXY" ]] && export http_proxy=http://$PROXY; export https_proxy=http://$PROXY;
+export SSL_CERT_FILE="$PWD/bins/cacert.pem"
+rm -rf /tmp/cookbooks
+pushd cookbooks
+find . -maxdepth 1 -type d '!' -name "*bcpc*" -a '!' -name "hannibal" -a '!' -name "hdfsdu" -a '!' -name '\.'| xargs rm -rf
+popd
+find ./cookbooks -name "Berksfile" -exec berks -b {} \; -exec berks vendor /tmp/cookbooks -b {} \; -exec rsync -ar /tmp/cookbooks . \;
```

Comment:
@kiiranh . What will happen if any of the community cookbook doesn't use Berkshelf? Also we may download more cookbooks if any of the external cookbooks have groups defined in Berksfile as in the case of [kafka-cookbook](https://github.com/mthssdrbrg/kafka-cookbook/blob/master/Berksfile#L5-L8) where a ``test`` group is defined which is supposed to be skipped using ``berks install --except test``.